pkg/cli/admin/mustgather/mustgather.go

```diff
@@ -282,7 +402,7 @@ func (o *MustGatherOptions) waitForPodRunning(pod *corev1.Pod) error {
 }
 
 func (o *MustGatherOptions) waitForGatherContainerRunning(pod *corev1.Pod) error {
-	return wait.PollImmediate(time.Second, 10*time.Minute, func() (bool, error) {
+	return wait.PollImmediate(time.Second, time.Minute, func() (bool, error) {
```

Comment:
Why we're changing this? This is a significant decrease from 10mins to 1min.

Reply:
I lowered this because I noticed that when a bad/non-existent image is specified the container remains in Waiting state while in an `ErrImagePull`/`ImagePullBackOff` loop and I wanted it to fail faster. I'll set it back to 10 minutes and check for the specific reason for waiting instead.